uefi/src/table/boot.rs

```diff
@@ -1139,6 +1139,30 @@ impl BootServices {
         assert_eq!((self.stall)(time), Status::SUCCESS);
     }
 
+    /// Adds, updates, or removes a configuration table entry
+    /// from the EFI System Table.
+    ///
+    /// # Safety
+    ///
+    /// This relies on `table_ptr` being allocated in the
+    /// pool of type `EfiRuntimeServicesData`.
+    ///
+    /// # Errors
+    ///
+    /// See section `EFI_BOOT_SERVICES.InstallConfigurationTable()` in the UEFI
+    /// Specification for more details.
+    ///
+    /// * [`uefi::Status::INVALID_PARAMETER`]
+    /// * [`uefi::Status::NOT_FOUND`]
+    /// * [`uefi::Status::OUT_OF_RESOURCES`]
+    pub unsafe fn install_configuration_table(
+        &self,
+        guid_entry: Guid,
+        table_ptr: *const c_void,
```

Comment:
Why are you using different names here compared to the definition above?

Reply:
I wanted to make sure we had proper names for the public API, the fn ptr is more of an impl detail, but I can sync the names as it's better for coherency.

Reply:
Now it's `entry_guid` vs `guid_entry`. reeee

Reply:
naming is hard, apologies for the back'n'forth